dbt/adapters/bigquery/python_submissions.py

```diff
@@ -64,6 +68,14 @@ def _get_job_client(
     def _submit_dataproc_job(self) -> dataproc_v1.types.jobs.Job:
         raise NotImplementedError("_submit_dataproc_job not implemented")
 
+    def _wait_operation(self, operation):
```

Comment:
do our existing tests cover this logic? do we need to add a unit test on this method?

Reply:
Yes there's existing integration test will cover it. I don't think a unit test is needed as we should be able to remove it once google fix their issue